src/nvim/buffer_defs.h

```diff
@@ -936,8 +934,8 @@ struct matchitem {
  * All row numbers are relative to the start of the window, except w_winrow.
  */
 struct window_S {
-  uint64_t handle;
-  int w_id;                         ///< unique window ID
+  handle_T handle;                  ///< unique identifier for the window
+
```

Comment:
might be rebase error in extmark then. I'll merge this, please push rebased extmark then and I'll look into it.